Задача 4. Даны 3 группы учеников плавания. Не соблюдается условие нормальности.
В 1 группе время на дистанцию 50 м составляют: 56, 60, 62, 55, 71, 67, 59, 58, 64, 67
Вторая группа : 57, 58, 69, 48, 72, 70, 68, 71, 50, 53
Третья группа: 57, 67, 49, 48, 47, 55, 66, 51, 54
Есть ли статистически значимые различия между группами?

In [1]:
import numpy as np
from scipy import stats

In [3]:
sample1 = np.array([56, 60, 62, 55, 71, 67, 59, 58, 64, 67])
sample2 = np.array([57, 58, 69, 48, 72, 70, 68, 71, 50, 53])
sample3 = np.array([57, 67, 49, 48, 47, 55, 66, 51, 54])

Принимаем α=0,05.

In [4]:
stats.kruskal(sample1, sample2, sample3)

KruskalResult(statistic=5.465564058257224, pvalue=0.0650380998590494)

Т.к. pvalue=0.0650380998590494 > α=0,05,  можно говорить о том что статистически значимых различий в группах учеников плавания нет.